contributing.md

```diff
@@ -17,7 +17,14 @@ feel free to propose changes to this document in a pull request.
 
 ## Translating
 
-👉 **Do not translate the content directly in this repository!**
+Here are some guidelines to keep in mind as a translator:
+
+- All translation happens on [Crowdin](https://crowdin.com/project/electron). **Do not open pull requests that translate the content in this repository!**
```

Comment:
This is better but still comes off as a little hostile. Maybe something like...

> * Don't open pull requests to translate Electron; instead, do all translations at [Crowdin](...)

?